jobs/settings/__init__.py

```diff
@@ -3,7 +3,9 @@
 import environ
 
 env = environ.Env(
-    DEBUG=(bool, False)
+    DEBUG=(bool, False),
+    ELASTIC_HOST_NAME=(str, 'localhost'),
```

Comment:
this is completely fine, but I normally don't use it like this. Normally I have the Env() method empty and then I add the casting in each var.
For example
`ELASTIC_HOST_PORT = env.str("ELASTIC_HOST_PORT", default="9200")`, but up to you.

Reply:
Oh, nice! I'm going to change it

Reply:
Done!